src/components/App/components/UserInput/UserInput.js

```diff
@@ -68,15 +70,22 @@ function UserInput(props) {
                 inputProps={{
                   id: 'font-size',
                   type: 'number',
-                  min: MIN_FONT_SIZE,
-                  max: MAX_FONT_SIZE,
+                  min: MIN_FONT_SIZE_IN_PX,
+                  max: MAX_FONT_SIZE_IN_PX,
```

Comment:
Another idea could be to show a message (where we currently show the error message) explaining why the preview is not changing. I'm not convinced that it's necessary. What do you think?